# drone geo fence for indoor flight
DroneGeoFence:
  x_min: -1.6
  x_max:  1.5
  y_min: -1.3
  y_max:  1.3
  z_min: -0.05
  z_max:  2.5